monai/transforms/utility/dictionary.py

```diff
@@ -1714,6 +1714,10 @@ class RandImageFilterd(MapTransform, RandomizableTransform):
             Probability the transform is applied to the data
         allow_missing_keys:
             Don't raise exception if key is missing.
+
+    Note:
+        - This method does not scale the image automatically. The user will need to manually scale
+          the image back to its proper size after RandImageFilterd transforms.
```

Comment:
```suggestion
    Note:
        - This transform does not scale output image values automatically to match the range of the input. 
          The output should be scaled by later transforms to match the input if this is desired.
```
The issue here is that the value range of the output doesn't match the input? This isn't a size problem since the input and output tensors have the same dimensions.